knowledge_base/bundle_variables/README.md

```diff
@@ -0,0 +1,65 @@
+# bundle_variables
+
+Example to show how bundle variables work.
+
+We will focus on the behavior of the variables section alone. To do so, we exclusively use
+the `validate` command, and use its JSON output mode to focus on the variables section.
+
```

Comment:
Are there any prerequisites worth mentioning? It might be a good standard just to list the supported CLI version for any KB?